### PR TITLE
Make Quarkus modules inherit from Infinispan parent

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.redhat.com/browse/ISPN-14733

### CODE REVIEW

File: quarkus/embedded/runtime/pom.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>org.jboss.narayana.jta</groupId>
             <artifactId>narayana-jta</artifactId>
-            <optional>true</optional>
+            <scope>compile</scope>
```
Comment (wburns) [inline quarkus/embedded/runtime/pom.xml:78]: Can just remove the line then?

**Review by wburns — APPROVED** [review]
Seems fine as far as I can see.

File: quarkus/embedded/runtime/pom.xml (hunk 1 above)
Comment (ryanemerson) [reply quarkus/embedded/runtime/pom.xml:78]: Without this I always get:

```java
[ERROR] /home/remerson/workspace/RedHat/infinispan/infinispan-alt/quarkus/embedded/runtime/src/main/java/org/infinispan/quarkus/embedded/runtime/graal/SubstituteTransactionClasses.java:[38,50] error: package com.arjuna.ats.jta does not exist
[ERROR] /home/remerson/workspace/RedHat/infinispan/infinispan-alt/quarkus/embedded/runtime/src/main/java/org/infinispan/quarkus/embedded/runtime/graal/SubstituteTransactionClasses.java:[46,33] error: package com.arjuna.ats.jta does not exist

```

Comment (wburns) [reply quarkus/embedded/runtime/pom.xml:78]: Ah, guess one of the poms must be setting the scope somewhere else :(

**Pull request merged**